tests/test_web_response.py

```diff
@@ -1171,3 +1173,36 @@ def test_text_is_json_encoded(self) -> None:
     def test_content_type_is_overrideable(self) -> None:
         resp = json_response({"foo": 42}, content_type="application/vnd.json+api")
         assert "application/vnd.json+api" == resp.content_type
+
+
+@pytest.mark.skipif(
+    not bool(os.environ.get("PYTHONASYNCIODEBUG")),
+    reason="Only run the test with debug mode",
+)
+async def test_warn_large_cookie(buf: Any, writer: Any) -> None:
+    resp = Response()
+    resp.set_cookie("foo", "8" * 4070, max_age=2600)  # No warning
+    req = make_request("GET", "/", writer=writer)
+
+    await resp.prepare(req)
+    await resp.write_eof()
+
+    cookie = re.search(b"Set-Cookie: (.*?)\r\n", buf).group(1)
+    assert len(cookie) == 4096
+    buf[:] = b""
```

Comment:
Why do you reuse a variable like this? Maybe create a new one?

Reply:
That was me, it comes from a fixture, so seemed weird to create a second one manually.
At the same time, I'm not really sure why there is a fixture just to create a `bytearray()` in the first place...

Reply:
Probably for reuse and consistency. Anyway, I don't think there should be more than one test in a test.